helpers/render.py

```diff
@@ -18,6 +19,27 @@
 from .colors import maintain_colors
 from .load_images import prepare_overlay_mask
 
+from numpngw import write_png
+import tifffile
+
+def convert_image_to_8bpc(image, bit_depth_output):
+    if bit_depth_output == 16:
+        image = image / 256
+        image = Image.fromarray(image.astype('uint8'))
+    elif bit_depth_output == 32:
+        image = image * 256
+        image = Image.fromarray(image.astype('uint8'))
+    return image
+
+def save_8_or_16bpc_image(image, outdir, filename, bit_depth_output):
```

Comment:
This function name does not indicate that the bit_depth can be saved with `32` as well

Reply:
Done!